src/Main/TextUserInterface.cpp

```diff
@@ -1542,7 +1564,7 @@ namespace VeraCrypt
 
 			// Re-encrypt volume header
 			SecureBuffer newHeaderBuffer (volume->GetLayout()->GetHeaderSize());
-			Core->ReEncryptVolumeHeaderWithNewSalt (newHeaderBuffer, volume->GetHeader(), options.Password, options.Pim,  options.Keyfiles);
+			Core->ReEncryptVolumeHeaderWithNewSalt (newHeaderBuffer, volume->GetHeader(), options.Password, options.Pim,  options.Keyfiles, true);
```

Comment:
Same as above: EMVOption parameter is always set to true regardless of the actual EMV option value.

Reply:
Same as above : VC in CLI mode always checks for EMV cards